lib/browser/highlight-tags.js

```diff
@@ -25,6 +25,22 @@ function highlight(js) {
     );
 }
 
+let highlightPolicy = {
+  createHTML: function(html) {
+    // The highlight function escapes its input.
+    return highlight(html);
+  }
```

Comment:
[Style] Nit: no need for this wrapper, I think?

```suggestion
  // The highlight function escapes its input.
  createHTML: highlight,
```